cmd/commands/root.go

```diff
@@ -0,0 +1,79 @@
+package command
+
+import (
+	"context"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	rookclient "github.com/rook/rook/pkg/client/clientset/versioned"
+	"github.com/spf13/cobra"
+	k8s "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	KubeConfig           string
+	KubeContext          string
+	OperatorNamespace    string
+	CephClusterNamespace string
+)
+
+// RootCmd represents the odf command
+var RootCmd = &cobra.Command{
+	Use:              "odf",
+	Short:            "Management and troubleshooting tools for ODF clusters.",
+	Aliases:          []string{"fdf"},
```

Comment:
How does the alias work on the root command? I thought we would have to rename the binary or have a symlink to get the `fdf` cli tool name.